BreakinIn/MatchmakerServer.cs

```diff
@@ -27,6 +27,7 @@ public class MatchmakerServer : AbstractEAServer
                 { "sele", typeof(SeleIn) }, //gets info for the current server
                 { "user", typeof(UserIn) }, //get my user info
                 { "onln", typeof(OnlnIn) }, //search for a user's info
+		{ "rept", typeof(ReptIn) }, //report user
```

Comment:
Indentation: should match up with the rest of the entries. Specifically - you are using tab indentation here, when all files in this project use space.